src/main/config/config.c

```diff
@@ -356,7 +355,11 @@ static void resetControlRateConfig(controlRateConfig_t *controlRateConfig) {
     controlRateConfig->tpa_breakpoint = 1500;
 
     for (uint8_t axis = 0; axis < FLIGHT_DYNAMICS_INDEX_COUNT; axis++) {
-        controlRateConfig->rates[axis] = 0;
+        if (axis == FD_YAW) {
```

Comment:
They were never equal. Even when set to the same value, they were not equal. Because of different curve scaling there was 10% difference between them. 
Personally I think lower yaw rate is safer. Because of requirements, pilot needs and hardware capabilities. Even my 250 class has yaw rate much lower than roll rate